### PR TITLE
Metrics implementation to push to Collector - #PLAT-888

### PR DESCRIPTION
**Pull request by gvelez17**
# Metrics implementation to push to Collector - #PLAT-888

## Description

Use same interface but push metrics to collector instead of using exporter

## How Has This Been Tested?

Describe the tests that you ran to verify your changes. Provide instructions for reproduction.

- [ ] simple test of service metrics package
- [ ] unit test for new util function

## Definition of Done

Before submitting this PR, please make sure:

- [x ] The work addresses the description and outcomes in the issue
- [ x] I have added relevant tests for new or updated functionality
- [ x] My code follows conventions, is well commented, and easy to understand
- [ x] My code builds and tests pass without any errors or warnings
- [x ] I have tagged the relevant reviewers
- [x ] I have updated the READMEs of affected packages
- [x ] I have made corresponding changes to the documentation
- [x ] The changes have been communicated to interested parties

## References:

Please list relevant documentation (e.g. tech specs, articles, related work etc.) relevant to this change, and note if the documentation has been updated.

### CODE REVIEW

File: src/services/anchor-service.ts [hunk 1]
```diff
@@ -18,11 +18,13 @@ import { RequestRepository } from '../repositories/request-repository.js'
 import { IpfsService } from './ipfs-service.js'
 import { EventProducerService } from './event-producer/event-producer-service.js'
 import { CeramicService } from './ceramic-service.js'
+import { ServiceMetrics as Metrics } from '../service-metrics.js'
 import { METRIC_NAMES } from '../settings.js'
 import { BlockchainService } from './blockchain/blockchain-service.js'
 import { inject, singleton } from 'tsyringe'
 import { CommitID, StreamID } from '@ceramicnetwork/streamid'
-import { Metrics } from '@ceramicnetwork/metrics'
+
```
Comment (stephhuynh18) [inline src/services/anchor-service.ts:26]: extra whitespace added

File: src/service-metrics.ts [hunk 2]
```diff
@@ -0,0 +1,91 @@
+/* Service metrics need to push to a collector rather than expose
```
Comment (stephhuynh18) [inline src/service-metrics.ts:1]: I wonder if this should live in the ceramic metrics package.

File: src/services/anchor-service.ts [hunk 3]
```diff
@@ -361,19 +366,25 @@ export class AnchorService {
...
       )
+      let before = Date.now()
       const anchor = await this._createAnchorCommit(candidate, i, ipfsProofCid, merkleTree)
       if (anchor) {
         anchors.push(anchor)
+        timings.push(Date.now() - before)
       }
     }
 
+    Metrics.recordAverage(METRIC_NAMES.TIME_ANCHOR_COMMITS_MS, timings)
```
Comment (stephhuynh18) [inline src/services/anchor-service.ts:387]: I am a little confused to why we are timing this

File: src/services/anchor-service.ts [hunk 4]
```diff
@@ -221,6 +223,9 @@ export class AnchorService {
     logger.debug('Persisting results to local database')
     const numAnchoredRequests = await this._persistAnchorResult(anchors, candidates)
 
+    // maybe here or somewhere in a loop get the time deltas for the successful anchors TODO
```
Comment (stephhuynh18) [inline src/services/anchor-service.ts:226]: I am a little confused about what we are trying to time here as well.

File: src/service-metrics.ts (hunk 2 above)
Comment (gvelez17) [reply src/service-metrics.ts:1]: Yeah it should live in its own package - which doesn't exist yet

Comment (gvelez17) [reply src/service-metrics.ts:1]: Yes if anything else ever needs this kind of metrics, we should make a repo just for these public metrics packages, I think - that way can quickly change them

File: src/services/anchor-service.ts (hunk 4 above)
Comment (gvelez17) [reply src/services/anchor-service.ts:226]: we'll pull out the timing and put it in a separate PR

File: src/services/anchor-service.ts (hunk 3 above)
Comment (gvelez17) [reply src/services/anchor-service.ts:387]: yeah will do a separate PR with timings and traces

**Review by stephhuynh18 — APPROVED** [review]
Once test pass and small comment

**Comment by gvelez17** [comment]
ugh unrelated checks are failing now?  tried merge from latest develop

**Pull request merged**